github.go

```diff
@@ -0,0 +1,267 @@
+package checkup
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+var errFileNotFound = fmt.Errorf("file not found on github")
+
+// GitHub is a way to store checkup results in a GitHub repository.
+type GitHub struct {
+	AccessToken     string `json:"access_token"`
+	RepositoryOwner string `json:"repository_owner"`
+	RepositoryName  string `json:"repository_name"`
+	CommitterName   string `json:"committer_name"`
+	CommitterEmail  string `json:"committer_email"`
+	Branch          string `json:"branch"`
+	Dir             string `json:"dir"`
```

Comment:
Some godoc on these fields would be useful, especially indicating that all of them are required.